content/blog/getting-started-to-generate-dscc-api-client-libraries-for-python-using-openapi-generator.md

```diff
@@ -0,0 +1,217 @@
+
+![](/img/openapi-generator-version-locatoin.png "Stable version for the openAPI generator project")
+
+In this blog, let's look at the painless and simplest application of the openAPI generator, that is using the JAR file. Actually using JAR file doesn't require installation at all. The JAR file can be downloaded and executed directly from the command lines. The requirement for executing the JAR file is that your workstation must be deployed with JAVA runtime 8 (JRE) at a minimum. 
+
+For more information about the deployment of JAVA runtime executables based on the operating system of your workstation, please take a look at the installation page from the JAVA website: <https://www.java.com/en/download/help/download_options.html>
+
+The JAR file for this openAPI generator is available at Maven.org. You can download it from the following location: <https://repo1.maven.org/maven2/org/openapitools/openapi-generator-cli/5.4.0/openapi-generator-cli-5.4.0.jar>. Below , you will find the syntax required to download the openAPI generator JAR files from the corresponding workstation:
+
+**For Mac/Linux users:**
+
+```shell
+~$ wget https://repo1.maven.org/maven2/org/openapitools/openapi-generator-cli/5.4.0/openapi-generator-cli-5.4.0.jar -O openapi-generator-cli.jar
+```
+
+**For Windows users:**
+
+```powershell
+> Invoke-WebRequest -OutFile openapi-generator-cli.jar https://repo1.maven.org/maven2/org/openapitools/openapi-generator-cli/5.4.0/openapi-generator-cli-5.4.0.jar
+```
+
+**Note:** The version of OpenAPI generator which is shown above is 5.4.0 (current version as of March 2022). Please keep in mind that in future, the location of the JAR file will change based on a new code version.  Please take a look at the figure above to obtain the latest version number, and modify the path to download the latest open-api-generator-cli JAR file.
+
+Once the JAR file is downloaded, you can execute the following CLI at the folder where the JAR file is downloaded to display the brief information on how to use this JAR file.
+
+```shell
+~$ java -jar openapi-generator-cli.jar help
+```
+
+The output will be something like this:
+
+```typescript
+usage: openapi-generator-cli <command> [<args>]
+
+The most commonly used openapi-generator-cli commands are:
+    author        Utilities for authoring generators or customizing templates.
+    batch         Generate code in batch via external configs.
+    config-help   Config help for chosen lang
+    generate      Generate code with the specified generator.
+    help          Display help information about openapi-generator
+    list          Lists the available generators
+    meta          MetaGenerator. Generator for creating a new template set and configuration for Codegen.  The output will be based on the language you specify, and includes default templates to include.
```

Comment:
Would it be possible to rephrase the last half of this sentence "and includes default templates to include." in order to not sound redundant? Maybe "and offers default templates to include."?

Reply:
Sorry that's the output from the help which unfortunately literally copied